lib/dmv_factory.rb

```diff
@@ -0,0 +1,25 @@
+class DmvFactory
+  attr_reader :office_list
+  def initialize
+    @office = nil
+  end
+
+  def create_offices_helper(api_data)
+    @office_list = api_data.map do |office|
+      @office = {
+       name: "#{office[:dmv_id]} #{office[:dmv_office]}", 
+       address: "#{office[:address_li]} #{office[:address__1]} #{office[:city]} 
+                 #{office[:state]} #{office[:zip]}", 
+       phone: office[:phone], 
+       services: office[:services_p]
+      }
+    end
+  end
```

Comment:
This works, and the rest of my comment is just something to think about:
Generally my advice is to avoid declaring/assigning instance variables outside of the `initialize` method when possible. Since this method is returning the value of `api_data.map...`, I wonder if you could access this collection by calling on the method directly instead.